### PR TITLE
1373-Getting-an-application-crash-when-pressing-the-Delete-key

### PR DESCRIPTION
**Pull request by giduac**
[Issue 1373-Getting-an-application-crash-when-pressing-the-Delete-key](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1373)
- `KT.CommonHelper.CheckContextMenuForShortcut()` handles direct type casts differently from .NET 8.0 onward. 
- Solution courtesy of @Tape-Worm
- And the change log

compile-results

### CODE REVIEW

File: Documents/Changelog/Changelog.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 =======
 
 ## 2024-11-xx - Build 2411 - November 2024
+* Resolved [#1373](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1373), `KT.CommonHelper.CheckContextMenuForShortcut()` handles direct type casts differently from .NET 8.0 onward. Solution courtesy of @Tape-Worm
```
Comment (Wagnerp) [inline Documents/Changelog/Changelog.md:6]: Please move to V85 patch section to reduce duplication, thanks!

Comment (giduac) [reply Documents/Changelog/Changelog.md:6]: @Wagnerp 
Where would that be?

Comment (Wagnerp) [reply Documents/Changelog/Changelog.md:6]: Should be line 136

**Comment by giduac** [comment]
@Wagnerp 
Done

**Pull request merged**